draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1513,6 +1531,7 @@ Constants:
 1.  c1 = -B / A
 2.  c2 = -1 / Z
 3.  c3 = sqrt(-Z^3)
```

Comment:
Does the sign of the root must be specified here or in the suite?

Reply:
No need to specify. Setting the sign of y gets the right result no matter which sign you choose for c3.

Reply:
That's true.